__tests__/lint/assertions-casing-kebab-case-error/.redocly.yaml

```diff
@@ -4,16 +4,26 @@ apis:
 
 rules:
   assert/operation-get:
-    context:
-      - type: Operation
-        matchParentKeys: [get]
-    subject: Operation
-    property: operationId
+    where:
+      - subject: 
+          type: Operation
+          filterInParentKeys: [get]
+          property: responses
+        assertions:
+          defined: true
+    subject: 
+      type: Operation
+      property: operationId
```

Comment:
same here, can we use `filterInParentKeys` directly here?

Reply:
Resolved